### ISSUE TITLE
Sites vocabulary

### ISSUE THREAD

**Issue by dxwell** [geological-survey-of-queensland/geological-properties-database#1]
Seeking input on the **Sites** vocabulary - a sub-set of the geological or administrative feature. Let's get a start on the list and then circulate to the wider audience.

@KellyVance @johnmckellar @GSQ-AI

Current contenders:
- borehole
- stream
- seismic line
- field site
- fossil site
- isotopic dating site
- petroleum resource site (or is this covered in **resource accumulation** in features vocab?)
- mine (with status: operating, care and maintenance, historical working, etc.)
- project site
- geological boundary
- base station
- sampling site

Other questionable sites (needs review):
- dykes
- sills
- faults
- shear zone
- fold
- joint
- fracture
- lineament
- trend
- trap
- dip

**Comment by johnmckellar** [comment]
Hi David,

Before we get into this, we, or, at least, I need to see where we are going with it, as there is a potential never-ending list of things to add, and, from my perspective of ignorance, different ways of approaching it all. Please see my comments below:

Current contenders:
•	Borehole  <<< a borehole has a site, but there are sampling sites (depths) in a borehole, for all sorts of things (including some listed below: e.g. fossils, isotopic dating). The borehole site at the surface may be a point, and the site at depth (for a sample), may not coincide (vertically) with the same point, if the hole is not vertical. And the sample site may be an interval
•	Stream  <<< site of a stream?? Also rivers, lakes, ocean, etc
•	seismic line
•	dykes <<< a point?? Or the 3D geometry of a dyke; also for veins, sills, etc
•	veins
•	sills
•	faults
•	shear zone
•	fold <<< 3D?? & there are all sorts of folds
•	joint <<< ditto & for fracture, etc
•	fracture
•	lineament
•	trend
•	dip
•	field site
•	fossil site
•	isotopic dating site
•	petroleum resource site (or is this covered in resource accumulation in features vocab?)
•	mine (with status: operating, care and maintenance, historical working, etc.)
•	project site
•	geological boundary
•	base station

Some of the things above (e.g. fault, fold, shear zone/deformation zone) will be covered by the Structures vocab and, presumably will be linked to the Structural Framework. How do we best approach this from a sites perspective?

Cheers,
John

**Comment by Greenwoodmatthew** [comment]
I agree with John, we need to define the purpose of site first.  I see site as a single location (X,Y,{+-}Z) compared to a survey which can be a polyline/polyon.  Things like a structural measurement of a structure (fold, fault, dyke...) is a field site but the feature is not a site.  Seismic survey is not a site, but a single shotpoint / CDP is a site (if we need to mention this?)  

I think this needs a fair bit of re-model before we are getting close to contenders

**Comment by ajtroup** [comment]
This echoes Matt a bit and I agree that the purpose needs to be defined. How are these things going to be used? What are they being defined for?

borehole - maybe, but most likely just the borehole at its origin point/wellhead. The downhole intervals and points are their own sites
stream - polyline, so not a single site. Are hydrological features even in our remit?
seismic line - not the line itself, but either shot point or cdps would be the sites (as Matt said)
field site - Yes, but depends on whether it is an area or a single sample/observation point
fossil site - yes, but theoretically this is a different kind of field site (being a field site at which fossils have been observed). Most will also have an area.
isotopic dating site - would be associated with a borehole or field site
petroleum resource site (or is this covered in resource accumulation in features vocab?) - would be tied to a borehole or a polygon (Though I'm not sure what this is intending to convey)
mine (with status: operating, care and maintenance, historical working, etc.) - polygon/area
project site - likely to be a polygon/area or be a "survey" of field sites
geological boundary - multiple sites form a boundary
base station - yes
sampling site - yes, but theoretically this is a different kind of field site
dykes - lithologic framework
sills - lithologic framework
faults - structural framework
shear zone - structural framework (also fault zone)
fold - structural framework
joint - structural framework (are we collecting data to this scale at all?)
fracture - structural framework
lineament - structrual framework
trend - structural framework
trap - structural framework
dip - this is an observation at a field site

**Comment by ajtroup** [comment]
This discussion stopped at some point and needs revisiting.

**Comment by KellyVance** [comment]
Instead of working top-down and trying to define every possible type of site, we need to work bottom up and just define the sites we deal with. Sites can be made up of other sites, sites can be single points in space, a traverse or line, an area, or a complete 3D object (i.e. point, line, polygon, volume).  Are our databases currently capturing geometry and location of sills, dykes, folds, etc? If yes, lets export that/those list(s). If no, forget them and move on. We can add stuff in when they become relevant. 

It also needs to be at the level that is _useful_ for indexing and finding information, not the highest resolution, most accurate. e.g. a CDP or shotpoint might be the most accurate representation of a seismic trace, but who cares? Outside analysing the actual SEG-Y or SEG-D themselves, we aren't finding information based on an individual shotpoint. We aren't storing individual traces in the database so why describe those seismic _sites_ as points.

Building on the previous lists my brain dump is below (indents indicate possible hierarchies or dependencies)

_Sites_
- borehole
  - wellbore
    - wellbore interval
- seismic survey area (as opposed to seismic survey, which is the act or surveying or sampling)
  - seismic line
- field site
  - fossil site
  - sampling site (i.e. take a physical sample such as soil, rock, water)
  - observation site (i.e. take a measurement such as strike and dip, XRF, colour)
- project location
- mine
- petroleum field
- base station
- geological site
  - basin extent
  - rock unit extent
  - structural feature
  - lithological feature

**Comment by KellyVance** [comment]
All (@dxwell @ajtroup @Greenwoodmatthew @johnmckellar @geoderekh @GSQ-AI )

Are the lithological and structural features mentioned above currently captured as sites in MERLIN? I believe they are treated as such by SGS for their detailed geology.

I'm going to stand up a rough draft vocabulary today or monday for SRA to develop their wireframes and UAT. It can always be added to as needed in the future, but i'd prefer to include known content from the start.

Also, if any of you can provide a one line definition for any of the concepts discussed in the context of a sites classification it would be greatly appreciated and save me a whole lot of time coming up with naive descriptions.

**Comment by johnmckellar** [comment]
To my knowlegde, we do not capture the geometry and location of sills, dykes, folds, etc? For faults, I believe we have data for major entities. I kind of had this discussion with Ross Lane when I was building the FOI vocab (here wrt structures).

However, according to Micaela, we do have data for all of the above for some areas of the Bowen Basin, courtesy of Renata & ?Joan, who went to a lot of trouble to try to get it, via GSQ and from ACARP, into the public domain. It presently sits with SGS, but requires some work (by SGS) to bring it into the system.  Micaela may wish to provide more info re this.

**Comment by Greenwoodmatthew** [comment]
From my understanding the current Surface Geology sites can have Lithological and Structural features added to the site (i.e. the stuctutral observations can be assicribed to a dyke etc.)

In this way (and going forward) the dyke or fault or whatever is not the site but the FOA that the site is of.  The measurements is an observation of the specific feature recorded at that site.  Dykes/faults/folds are features, not sites, yes?

@CourteneyD knows so much more about MERLIN SG data.  Tagging so she can tell me i am wrong

**Comment by KellyVance** [comment]
@Greenwoodmatthew @CourteneyD 
I think you're right. The lithological and structural features are their own discrete _components_ of larger features and therefore ultimate features in their own right.
As opposed to something like a field sampling location which _represents_ a larger feature and is therefore a proximate feature (site).

Result: All the geological sites listed above should ditched out of the sites vocab and should be incorporated into the GeoFeatures Ontology and Vocab at some point in the near-but-not-immediate future

**Comment by CourteneyD** [comment]
Nope you are right - they are just features of the site.

@Fred-v-G - tagging you to see this conversation
